modules/cloud-provider/hetzner.nix

```
@@ -0,0 +1,50 @@
+{ config, lib, ... }:
+with lib;
+let cfg = config.lounge-rocks.cloud-provider.hetzner;
+
+in {
+
+  options.lounge-rocks.cloud-provider.hetzner = {
+    enable = mkEnableOption "activate hetzner";
```

Verb get's autopopulated by `mkEnableOption` for docs
```suggestion
    enable = mkEnableOption "hetzner configuration";
```